records_versioning/signals.py

```
@@ -53,10 +55,32 @@
 @receiver(post_save, sender='diab_medications.MedicationOrder')
 def medication_order_saved(sender: object, instance: object, created: bool, **kwargs: object) -> None:
     """Signal receiver for post_save on MedicationOrder model to create a version."""
-    _create_version_on_save(instance)
+    _create_version_on_save(instance)
+    try:
+        if created:
+            AISummary.objects.create(
+                patient=instance.patient,
+                content_type=ContentType.objects.get_for_model(instance.__class__),
+                object_id=str(instance.pk),
+                summary=f"Medication {getattr(instance, 'name', '')} {getattr(instance, 'dose', '')} {getattr(instance, 'frequency', '')}"
```

Ruff (0.12.2)

Line too long (138 > 88)

(E501)